frameworks/hdfs/universe/marathon.json.mustache

```diff
@@ -72,30 +72,30 @@
     "MESOS_HTTP_AUTHENTICATEE": "com_mesosphere_dcos_http_Authenticatee",
     {{/service.service_account_secret}}
 
-    {{#service.tls.enabled}}
-    "TLS_ENABLED": "{{service.tls.enabled}}",
-    "TASKCFG_ALL_TLS_ENABLED": "{{service.tls.enabled}}",
+    {{#service.security.tls.enabled}}
+    "TLS_ENABLED": "{{service.security.tls.enabled}}",
+    "TASKCFG_ALL_TLS_ENABLED": "{{service.security.tls.enabled}}",
     "TASKCFG_ALL_NAME_NODE_HTTPS_PORT":"{{hdfs.name_node_https_port}}",
     "TASKCFG_ALL_JOURNAL_NODE_HTTPS_PORT":"{{hdfs.journal_node_https_port}}",
     "TASKCFG_ALL_DATA_NODE_HTTPS_PORT":"{{hdfs.data_node_https_port}}",
-    {{/service.tls.enabled}}
+    {{/service.security.tls.enabled}}
 
     {{#journal_node.enable_readiness_check}}
     "JOURNAL_READINESS_CHECK_ENABLED": "{{journal_node.enable_readiness_check}}",
     "TASKCFG_ALL_JOURNAL_READINESS_CHECK_ENABLED": "{{journal_node.enable_readiness_check}}",
     {{/journal_node.enable_readiness_check}}
 
 
-    {{#service.kerberos.enabled}}
-    "KERBEROS_ENABLED": "{{service.kerberos.enabled}}",
-    "TASKCFG_ALL_KERBEROS_ENABLED": "{{service.kerberos.enabled}}",
-    "TASKCFG_ALL_KDC_HOST_NAME": "{{service.kerberos.kdc_host_name}}",
-    "TASKCFG_ALL_KDC_HOST_PORT": "{{service.kerberos.kdc_host_port}}",
-    "KEYTAB_SECRET_PATH": "{{service.kerberos.keytab_secret}}",
-    "TASKCFG_ALL_KERBEROS_PRIMARY": "{{service.kerberos.primary}}",
-    "TASKCFG_ALL_KERBEROS_PRIMARY_HTTP": "{{service.kerberos.primary_http}}",
-    "TASKCFG_ALL_KERBEROS_REALM": "{{service.kerberos.realm}}",
-    {{/service.kerberos.enabled}}
+    {{#service.security.kerberos.enabled}}
+    "KERBEROS_ENABLED": "{{service.security.kerberos.enabled}}",
+    "TASKCFG_ALL_KERBEROS_ENABLED": "{{service.security.kerberos.enabled}}",
+    "TASKCFG_ALL_KDC_HOST_NAME": "{{service.security.kerberos.kdc_host_name}}",
+    "TASKCFG_ALL_KDC_HOST_PORT": "{{service.security.kerberos.kdc_host_port}}",
+    "KEYTAB_SECRET_PATH": "{{service.security.kerberos.keytab_secret}}",
```

Comment:
Minor nit. Could we at least rename this environment variable to remove the `_PATH`? Ideally, we could have the same environment variables as the other services, but I can address this as part of #1871.